tests/experimental/datasources/test_pandas_datasource.py

```diff
@@ -48,7 +54,7 @@ def test_construct_csv_asset_directly(csv_path):
     asset = CSVAsset(
         name="csv_asset",
         path=csv_path,
-        regex=r"yellow_tripdata_sample_(\d{4})-(\d{2}).csv",
+        regex=cast(Pattern, r"yellow_tripdata_sample_(\d{4})-(\d{2}).csv"),
```

Comment:
This eliminates `PyCharm` warning.

Reply:
It may eliminate the PyCharm warning but it's incorrect.